client/src/app/ui/modules/choice-dialog/components/choice-dialog/choice-dialog.component.html

```diff
@@ -6,12 +6,22 @@ <h2 mat-dialog-title *ngIf="data">{{ data.title | translate }}</h2>
     <!-- Content -->
     <div mat-dialog-content *ngIf="data?.choices">
         <os-list-search-selector
+            *ngIf="!data.sortFn"
             ngDefaultControl
             formControlName="select"
             [multiple]="data.multiSelect!"
             [inputListValues]="data.choices!"
             (selectionChanged)="onSelectionChanged($event)"
         ></os-list-search-selector>
+        <os-list-search-selector
+            *ngIf="data.sortFn"
+            ngDefaultControl
+            formControlName="select"
+            [multiple]="data.multiSelect!"
+            [sortFn]="data.sortFn"
+            [inputListValues]="data.choices!"
+            (selectionChanged)="onSelectionChanged($event)"
+        ></os-list-search-selector>
```

Comment:
Can't you combine this into one tag? E.g. by changing the list-search-selector to be able to handle an undefined sort function?